info/v2/container.go

```diff
@@ -74,9 +74,10 @@ type ContainerSpec struct {
 	Memory    MemorySpec `json:"memory,omitempty"`
 
 	// Following resources have no associated spec, but are being isolated.
-	HasNetwork    bool `json:"has_network"`
-	HasFilesystem bool `json:"has_filesystem"`
-	HasDiskIo     bool `json:"has_diskio"`
+	HasNetwork       bool `json:"has_network"`
+	HasFilesystem    bool `json:"has_filesystem"`
+	HasDiskIo        bool `json:"has_diskio"`
+	HasCustomMetrics bool `json:"has_custommetrics"`
```

Comment:
We are going to add CustomMetricsSpec later though? With metric name and type, so metrics could be discovered.